rpcs3/Emu/RSX/Overlays/overlay_osk.cpp

```diff
@@ -543,7 +590,14 @@ namespace rsx
 					{
 						w--;
 
-						if (!c.outputs.empty())
+						std::vector<std::u32string> outputs;
+
+						if (m_selected_mode < layer_mode::mode_count < c.outputs.size())
+						{
+							outputs = c.outputs[m_selected_mode];
+						}
```

Comment:
See comment for similar block.

Reply:
Fixed I guess.
Also fixed the nasty typo in the if condition